### PR TITLE
feat:(design-system): Use `input-group` from `XDS`

### PR DESCRIPTION
**Pull request by tajespasarela**
EX-7943

### CODE REVIEW

File: src/components/search-box.vue [hunk 1]
```diff
@@ -1,26 +1,27 @@
 <template>
-  <div class="x-input-group x-input-group--card x-border-color--lead">
+  <div class="x-input-group x-input-group-lead x-rounded-sm">
     <SearchInput
       :autofocus="false"
       :placeholder="$t('searchBox.placeholder')"
-      :class="$x.device === 'mobile' ? 'x-padding--left-05' : 'x-padding--left-08'"
+      :instant="false"
+      class="desktop:x-pl-40"
```
Comment (CachedaCodes) [inline src/components/search-box.vue:7]: The input group's first child now has a margin of `16px`, so now the input is 16px more to the right than it should be in desktop (in mobile is ok).

```suggestion
      class="desktop:x-pl-24"
```

File: src/components/search-box.vue [hunk 2]
```diff
@@ -1,26 +1,27 @@
 <template>
-  <div class="x-input-group x-input-group--card x-border-color--lead">
+  <div class="x-input-group x-input-group-lead x-rounded-sm">
     <SearchInput
```
Comment (CachedaCodes) [inline src/components/search-box.vue:3]: Font size has changed from `16px` to inheriting `14px`, I think we should change it in the home template from `x-archetype-utils` so they're the same.

**Review by CachedaCodes — CHANGES_REQUESTED** [review]
Nice 👍

File: src/components/search-box.vue [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
     <ClearSearchInput
       v-if="$x.device === 'desktop' && $x.query.searchBox"
-      class="x-button-lead !x-button-sm x-button-ghost x-max-h-32"
+      class="x-input-group-button"
     >
       {{ $t('searchBox.clear') }}
     </ClearSearchInput>
 
-    <button
+    <SearchButton
```
Comment (CachedaCodes) [inline src/components/search-box.vue:17]: The search button is now bigger than the one in the `home-template` so the same as above with the font

File: src/components/search-box.vue [hunk 4]
```diff
@@ -1,26 +1,27 @@
 <template>
-  <div class="x-input-group x-input-group--card x-border-color--lead">
+  <div class="x-input-group x-input-group-lead x-rounded-sm">
```
Comment (ajperezbau) [inline src/components/search-box.vue:2]: I don't know if the designs are updated, but there the input group has the neutral 90 color.

```suggestion
  <div class="x-input-group x-rounded-sm">
```

**Review by ajperezbau — COMMENTED** [review]
The empathize is rendering a double scroll on mobile, this will be solved when [this PR](https://github.com/empathyco/x-archetype/pull/262) is merged.

Should we wait till that to merge this one?

Looks good to me apart from the comments of @CachedaCodes 👌

File: src/components/search-box.vue (hunk 4 above)
Comment (tajespasarela) [reply src/components/search-box.vue:2]: the designs we have to match are these of the XDS: https://www.figma.com/file/9a1DMgqIHt5ZZBvS2waaOs/X-DS?node-id=4126%3A81277&t=mcdh9xFjIwKM0MlH-0

File: src/components/search-box.vue (hunk 1 above)
Comment (tajespasarela) [reply src/components/search-box.vue:7]: fixed

File: src/components/search-box.vue (hunk 3 above)
Comment (tajespasarela) [reply src/components/search-box.vue:17]: I don't see it bigger. It is 40px x 40px in both cases. Also the icon is 24px in both cases

File: src/components/search-box.vue (hunk 2 above)
Comment (tajespasarela) [reply src/components/search-box.vue:3]: done in this other PR: https://github.com/empathyco/x/pull/1079

**Review by ajperezbau — APPROVED** [review]

**Review by CachedaCodes — APPROVED** [review]

**Pull request merged**